streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -324,14 +316,12 @@ private List<KeyValue<Long, Long>> getAllRecordPerKey(final Long key, final List
         return recordsPerKey;
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
-    public void shouldBeAbleToPerformMultipleTransactions(final String eosConfig) throws Exception {
+    @Test
+    public void shouldBeAbleToPerformMultipleTransactions() throws Exception {
         final StreamsBuilder builder = new StreamsBuilder();
         builder.stream(SINGLE_PARTITION_INPUT_TOPIC).to(SINGLE_PARTITION_OUTPUT_TOPIC);
 
         final Properties properties = new Properties();
-        properties.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, eosConfig);
```

Comment:
Given that this is `EosIntegerationTest` we should run with `EXACTLY_ONCE_V2`, right?